engine/runtime-integration-tests/src/test/java/org/enso/compiler/test/StaticAnalysisTest.java

```diff
@@ -39,8 +40,8 @@ public abstract class StaticAnalysisTest {
           (builder) ->
               builder
                   .option(RuntimeOptions.ENABLE_STATIC_ANALYSIS, "true")
-                  .option(RuntimeOptions.LOG_LEVEL, Level.INFO.getName())
-                  .option(RuntimeOptions.LOG_LEVEL, Level.SEVERE.getName())
+                  .option(RuntimeOptions.LOG_LEVEL, Level.FINE.getName())
+                  .logHandler(JulHandler.get())
```

Comment:
Some tests are adding this `logHandler` and some don't: 
- How does one find out when to add it?
- Shouldn't it always be added by default?